joe.go

```diff
@@ -41,6 +41,17 @@ func availableFiles() (a []string, err error) {
 			availableGitignores = append(availableGitignores, strings.Replace(f.Name(), ".gitignore", "", 1))
 		}
 	}
+
+	files, err = ioutil.ReadDir(path.Join(dataPath, "Global"))
```

Comment:
I think we should generalize this instead of searching in just `Global`. We should look at all sub-dirs and recursively search in them.